interop/test_utils.go

```diff
@@ -454,6 +454,41 @@ func DoCancelAfterFirstResponse(tc testpb.TestServiceClient) {
 	}
 }
 
+// DoStatusCodeAndMessage checks that the status code is propagated back to the client.
+func DoStatusCodeAndMessage(tc testpb.TestServiceClient) {
+	var code int32 = 2
```

Comment:
I could do that on line 461.  However, I can't seem to make it work on line 463; when I try, the compiler says this:

interop/test_utils.go:463: cannot use codes.Unknown (type codes.Code) as type *int32 in field value

Since I'd like both lines to use the same value, it seemed better to define an int32 and use it in both places.